public/app/features/dashboard/dashgrid/DashboardEmpty.tsx

```diff
@@ -0,0 +1,221 @@
+            disabled={!canCreate}
+          >
+            Add visualization
+          </Button>
+        </div>
+        <div className={cx(styles.centeredContent, styles.others)}>
+          <div className={cx(styles.containerBox, styles.centeredContent, styles.rowContainer)}>
+            <div className={cx(styles.headerSection, styles.headerSmall)}>Add a row</div>
+            <div className={cx(styles.bodySection, styles.bodySmall)}>
+              Group your visualizations into expandable sections.
+            </div>
+            <Button
+              icon="plus"
+              fill="outline"
+              aria-label="Add new row"
+              onClick={() => {
+                reportInteraction('Create new row');
+                onCreateNewRow();
+              }}
+              disabled={!canCreate}
+            >
+              Add row
+            </Button>
+          </div>
+          <div className={cx(styles.containerBox, styles.centeredContent, styles.libraryContainer)}>
+            <div className={cx(styles.headerSection, styles.headerSmall)}>Import panel</div>
+            <div className={cx(styles.bodySection, styles.bodySmall)}>
+              Import visualizations that are shared with other dashboards.
+            </div>
+            <Button
+              icon="plus"
+              fill="outline"
+              aria-label="Add new panel from panel library"
+              onClick={() => {
+                reportInteraction('Add a panel from the panel library');
+                onAddLibraryPanel();
+              }}
+              disabled={!canCreate}
+            >
+              Import library panel
+            </Button>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    wrapper: css({
+      label: 'dashboard-empty-wrapper',
+      flexDirection: 'column',
+      maxWidth: '920px',
+      gap: theme.spacing.gridSize * 4,
+    }),
+    containerBox: css({
+      label: 'container-box',
+      flexDirection: 'column',
+      boxSizing: 'border-box',
+      border: '1px dashed rgba(110, 159, 255, 0.5)',
+    }),
+    centeredContent: css({
+      label: 'centered',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+    }),
+    visualizationContainer: css({
+      label: 'visualization-container',
+      padding: theme.spacing.gridSize * 4,
+    }),
+    others: css({
+      label: 'others-wrapper',
+      alignItems: 'stretch',
+      flexDirection: 'row',
+      gap: theme.spacing.gridSize * 4,
+    }),
+    rowContainer: css({
+      label: 'row-container',
+      padding: theme.spacing.gridSize * 3,
+    }),
+    libraryContainer: css({
+      label: 'library-container',
+      padding: theme.spacing.gridSize * 3,
+    }),
+    visualizationContent: css({
+      gap: theme.spacing.gridSize * 2,
+    }),
+    headerSection: css({
+      label: 'header-section',
+      fontWeight: 600,
+    }),
```

Comment:
```suggestion
    headerSection: css({
      label: 'header-section',
      fontWeight: 600,
        textAlign: 'center',
    }),
```

looks better for responsive

Reply:
yes! thank you